setup.py

```diff
@@ -1,39 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
This file is needed by `make pkg` command:
```bash
_info "virtual environment: installing plugin package"
$PYTHON_ENV/bin/pip install -I ${__proj_dir}
```

without this file, ACI package generation is impossible.